### PR TITLE
Move VocLoader to Mods.D2k and shp conversion commands to Mods.Cnc

### PR DESCRIPTION
**Pull request by reaperrr**
Split from #17355.

### CODE REVIEW

**Comment by Mailaender** [comment]
You need to adjust the namespace for `OpenRA.Mods.D2k/AudioLoaders/VocLoader.cs`.

**Comment by reaperrr** [comment]
Fixed.

**Comment by GraionDilach** [comment]
Not fixed - you added it into Cnc namespace while having it at a D2k folder.

**Comment by reaperrr** [comment]
Fixed.

**Review by Mailaender — DISMISSED** [review]

File: OpenRA.Mods.Common/UtilityCommands/Glob.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 namespace OpenRA.Mods.Common.UtilityCommands
 {
-	static class Glob
+	public static class Glob
```
Comment (teinarss) [inline OpenRA.Mods.Common/UtilityCommands/Glob.cs]: Wouldnt it make sense to move this too? Its only used by `ConvertPngToShpCommand`

**Comment by reaperrr** [comment]
Updated.

**Review by abcdefg30 — APPROVED** [review]

**Review by teinarss — APPROVED** [review]

**Comment by teinarss** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/a345abc17de5cd3cb67fa012d1ac342a125cc857...eedbe08bd2e408236249491a642c18edaf43c67c

**Comment by evgeniysergeev** [comment]
Wait. Why you doing that?
I’m trying to remove all dependencies from d2 to d2k and depend only on Mod.Common,
But you move VocLoader to d2k mod, and now I need to always use d2k in d2.
Can this be rolled back?

**Comment by reaperrr** [comment]
In that case, since D2k doesn't use VocLoader at all (only aud and wav), rather than reverting that part, wouldn't it make more sense to remove VocLoader from upstream completely and move it to https://github.com/OpenRA/d2/tree/master/OpenRA.Mods.D2 instead?

Voc is even more D2-specific than Wsa, which isn't upstream, either, and I highly doubt any mod other than D2-based ones will ever use it.

**Comment by evgeniysergeev** [comment]
In the past, this issue was created to move VocLoader from d2 to OpenRA https://github.com/OpenRA/d2/issues/89
Now need to move it back?

**Comment by pchote** [comment]
This PR was working towards the goal of not having *any* mod-specific formats in Mods.Common, which I think is a good idea - why should something like KKnD be forced to include code for Voc, Mix, VQA, etc.

We could put the VocLoader in Mods.Cnc - Pak is already there, after all.

**Comment by evgeniysergeev** [comment]
Just let me know what to do with d2 mod: copy VocLoader and probably Pak, or use Mods.Cnc and keep it in dependecies

**Comment by reaperrr** [comment]
> D2VocLoader should be called VocLoader because its not a dune2 specific format. Its actualy a standard Creative VOC format multiple old rts use (example: Warcraft 1), and it might be a good thing to have this decoupled from D2 branding.

Hm, I wasn't aware of that.
@pchote what would be your preference, shift it to Mods.Cnc as you suggested, make an exception for Voc due to the above (revert it back to Mods.Common), or move both Voc and Pak out of OpenRA to D2?

**Comment by pchote** [comment]
IMO Mods.Cnc is the simplest solution.

**Pull request merged**